cirq/_doc.py

```diff
@@ -54,3 +54,33 @@ def document(value: Any, doc_string: str = ''):
         # print(traceback.format_stack(limit=2)[0])
         pass
     return value
+
+
+# The below is based on tools/tensorflow_docs/api_generator/doc_controls.py#L295
+# in the https://github.com/tensorflow/docs repo
```

Comment:
Optional nit: I'd better to provide direct link to a commit. As it is, the reader is required to do a small copy-paste dance to get there.

Reply:
Agreed - that's how I started, but then the linter complained ... hmm I think there might be a way - if the line starts with `http*`...let me check.

Reply:
we should start keeping a list of times when the linter makes us make things less readable

Reply:
@mpharrigan added a "had-to-fight-with-the-linter" label :) Feel free to use it